sdk/freertos_app_cpu0/src/FreeRTOSConfig.h

```diff
@@ -1,247 +1,251 @@
+        while (1)                                                                                                      \
+            ;
+
+/* If configTASK_RETURN_ADDRESS is not defined then a task that attempts to
+return from its implementing function will end up in a "task exit error"
+function - which contains a call to configASSERT().  However this can give GCC
+some problems when it tries to unwind the stack, as the exit error function has
+nothing to return to.  To avoid this define configTASK_RETURN_ADDRESS to 0.  */
+#define configTASK_RETURN_ADDRESS NULL
+
+/****** Hardware specific settings. *******************************************/
+
+/*
+ * The application must provide a function that configures a peripheral to
+ * create the FreeRTOS tick interrupt, then define configSETUP_TICK_INTERRUPT()
+ * in FreeRTOSConfig.h to call the function.  This file contains a function
+ * that is suitable for use on the Zynq MPU.  FreeRTOS_Tick_Handler() must
+ * be installed as the peripheral's interrupt handler.
+ */
+void vConfigureTickInterrupt(void);
+#define configSETUP_TICK_INTERRUPT() vConfigureTickInterrupt()
+
+void vClearTickInterrupt(void);
+#define configCLEAR_TICK_INTERRUPT() vClearTickInterrupt()
+
+/* The following constant describe the hardware, and are correct for the
+Zynq MPU. */
+#define configINTERRUPT_CONTROLLER_BASE_ADDRESS         (XPAR_PS7_SCUGIC_0_DIST_BASEADDR)
+#define configINTERRUPT_CONTROLLER_CPU_INTERFACE_OFFSET (-0xf00)
+#define configUNIQUE_INTERRUPT_PRIORITIES               32
+
+/****** Network configuration settings - only used when the lwIP example is
+built.  See the page that documents this demo on the http://www.FreeRTOS.org
+website for more information. ***********************************************/
+
+/* The priority for the task that unblocked by the MAC interrupt to process
+received packets. */
+#define configMAC_INPUT_TASK_PRIORITY (configMAX_PRIORITIES - 1)
+
+/* The priority of the task that runs the lwIP stack. */
+#define configLWIP_TASK_PRIORITY (configMAX_PRIORITIES - 2)
+
+/* The priority of the task that uses lwIP sockets to provide a simple command
+line interface. */
+#define configCLI_TASK_PRIORITY (tskIDLE_PRIORITY)
+
+/* MAC address configuration. */
+#define configMAC_ADDR0 0x00
+#define configMAC_ADDR1 0x13
+#define configMAC_ADDR2 0x14
+#define configMAC_ADDR3 0x15
+#define configMAC_ADDR4 0x15
+#define configMAC_ADDR5 0x16
+
+/* IP address configuration. */
+#define configIP_ADDR0 172
+#define configIP_ADDR1 25
+#define configIP_ADDR2 218
+#define configIP_ADDR3 200
+
+/* Netmask configuration. */
+#define configNET_MASK0 255
+#define configNET_MASK1 255
+#define configNET_MASK2 255
+#define configNET_MASK3 0
+
+/* MS to tick macros */
+#define pdMS_TO_TICKS(xTimeInMs)    ((TickType_t) (((xTimeInMs) * (uint64_t) configTICK_RATE_HZ) / (uint64_t) 1000U))
+#define pdTICKS_TO_MS(xTimeInTicks)    (((xTimeInTicks) * (uint64_t) 1000U) / (double) configTICK_RATE_HZ)
+
```

Comment:
@Known4225 

Can you confirm that these added lines are the only changes to this file? The diff shows that the whole file was changed, why? It looks like this is the only addition.